.github/workflows/nav-veileders-flate-deploy-poao-frontend-dev.yaml

```diff
@@ -5,6 +5,8 @@ on:
     paths:
       - '.github/workflows/nav-veileders-flate-deploy-poao-frontend-dev.yaml'
       - apps/nav-veileders-flate/nais/nais-dev.yaml
+    branches:
+      - main
```

Comment:
litt grensetilfelle, men nå er det endring i nais-fila, og da får ikke vi testet det i pr-bygget siden denne også ventes med til vi pusher. Men samtidig litt rart å kjøre det før et er godkjent. så kan ev pushe for å teste koden nå

Reply:
Det er nok best å unngå at PRer deployer serverkoden til dev automatisk. I de tilfellene man trenger å teste endringer på poao-frontend-config er det nok best å deploye det manuelt.